modules/apps/object/object-test/src/testFunctional/tests/macros/ObjectAdmin.macro

```diff
@@ -10,6 +10,27 @@ definition {
 		Click(locator1 = "ObjectAdmin#SAVE_CUSTOM_OBJECT");
 	}
 
+	macro deleteAllCustomObjects {
+		ObjectAdmin.openObjectAdmin();
+
+		if (IsElementNotPresent(locator1 = "ObjectAdmin#VIEW_CUSTOM_OBJECT")) {
+			ObjectAdmin.notViewSystemObject();
+		}
+		else {
+			while (IsElementPresent(locator1 = "ObjectAdmin#VIEW_CUSTOM_OBJECT")) {
+				Click(locator1 = "ObjectAdmin#SELECT_FIRST_CUSTOM_OBJECT");
+
+				Click(locator1 = "ObjectAdmin#DELETE_CUSTOM_OBJECT");
+
+				VerifyElementPresent(locator1 = "ObjectAdmin#MESSAGE_SUCCESS");
+			}
+		}
+	}
+
+	macro notViewSystemObject {
+		AssertElementNotPresent(locator1 = "ObjectAdmin#VIEW_CUSTOM_OBJECT");
+	}
```

Comment:
You can remove this macro, there is no need for this.

Reply:
@rodrigocunhaa Was removed.